src/App.js

```diff
@@ -1,15 +1,25 @@
-import React, { useCallback, useState } from 'react';
+import { Switch, Route } from 'react-router';
+import ChatRoom from './components/ChatRoom';
+import ChatRoomList from './components/ChatRoomList';
+import FriendList from './components/FriendList';
+import FriendProfile from './components/FriendProfile';
+import Login from './components/Login';
+import Page404 from './components/Page404';
+import ContactsProvider from './contexts/ContactsProvider';
+import RoomsProvider from './contexts/RoomsProvider';
+import Settings from './components/Settings';
+import Home from './components/Home';
 import styled from 'styled-components';
-import ActiveUser from './components/ActiveUser';
-import ChatContainer from './components/ChatContainer';
-import ChatForm from './components/ChatForm';
+import Navbar from './components/Navbar';
+import Signup from './components/Signup';
+import RequireLogin from './components/RequireLogin';
```

Comment:
```suggestion
import LoginRequired from './components/LoginRequired';
```

Wrapper 컴포넌트는 형용사로 작성하는 경우가 많더라구요~!